Add Storytel provider documentation (#701)
music-assistant/music-assistant.io: src/content/docs/music-providers/storytel.md

---
title: "Storytel"
---

# Storytel <img src="/assets/icons/storytel.svg" alt="Preview image" style="width: 70px; float: right;" loading="lazy" />

> Music Assistant has support for [Storytel](https://www.storytel.com). In some regions the service is also known as [Mofibo](https://www.mofibo.com). This provider will allow streaming of audiobooks and podcasts from one of the world's largest subscription listening services. This provider is contributed and maintained by [Jonas Pedersen](https://github.com/jonasbp2011)

Reviewer: ```suggestion
Music Assistant has support for [Storytel](https://www.storytel.com), which in some countries is sold as [Mofibo](https://www.mofibo.com). Contributed and maintained by [Jonas Pedersen](https://github.com/jonasbp2011)

Storytel is a Swedish subscription service for audiobooks and podcasts, and one of the largest of its kind. A monthly fee covers unlimited listening rather than buying titles one at a time, and the catalogue is strongest in the Nordic languages alongside the usual English titles.

Connecting your account puts your Storytel library inside Music Assistant, with the catalogue there to search. Where you got to in a book travels in both directions, so a title started in the Storytel app can be picked up again in Music Assistant, or the other way round.
```

---
title: "Storytel"
---

> # Storytel (Mofibo) <img src="/assets/icons/storytel.svg" alt="Preview image" style="width: 70px; float: right;" loading="lazy" />

Reviewer: ```suggestion
# Storytel <img src="/assets/icons/storytel.svg" alt="Preview image" style="width: 70px; float: right;" loading="lazy" />
```
You mention the Mofibo connection below.

> Music Assistant has support for [Storytel](https://www.storytel.com). In some regions the service is also known as [Mofibo](https://www.mofibo.com).

Reviewer: ```suggestion
Music Assistant has support for [Storytel](https://www.storytel.com). In some regions the service is also known as [Mofibo](https://www.mofibo.com). This provider will allow streaming of audiobooks and podcasts from one of the world's largest subscription listening services. This provider is contributed and maintained by [Jonas Pedersen](https://github.com/jonasbp2011)
```

| Self-Hosted Local Media                           |          No          |
| Media Types Supported                             | Audiobooks, Podcasts |
| [Recommendations](/ui/#view---discover) Supported |         Yes          |
| Lyrics Supported                                  |          No          |
| [Radio Mode](/ui/#track-menu)                     |          No          |
| Maximum Stream Quality                             |   Lossy, AAC/MP4     |
| Login Method                                      |       Password       |

### Other

- Storytel supports both audiobooks and podcasts from the user's library.
- Search is available for both audiobooks and podcasts.
- Personalized recommendations are supported.
- Chapter navigation.
- Listening progress is automatically synced back to Storytel through bookmarks.
- Library management is supported for adding and removing audiobooks and podcasts.
- The provider can be configured with one or more content languages to influence search and recommendations.

### Configuration

To set up the Storytel provider, follow these steps:

1. Navigate to `Settings`.

> 2. Under Music Sources, click `Add a music source`, select `Storytel (Mofibo)`, and fill in your `Username or email` and `Password`.

Reviewer: ```suggestion
2. Under Music Sources, click `Add a music source`, select `Storytel`, and fill in your `Username or email` and `Password`.
```

3. Select the content languages you want to use for search and recommendations.
4. Click `Save`.

### Settings

In addition to `Username or email` and `Password`, there is also:

- <b>Content Languages.</b> Select which languages to include in recommendations and search results. Multiple languages can be selected.

## Known Issues / Notes

> - Storytel is still beta in Music Assistant. Please report any issues or unexpected behaviour.

Reviewer: ```suggestion
```

Not required

> - Browse support is not currently implemented for Storytel. Instead, use the audiobook and podcast libraries to access your media.

Reviewer: Search is also available?

Author: Yes, should this be added to the notes here?

Reviewer: Yes I think it is just a matter if a couple of words in the second sentence since you are describing what you can do in lieu of browse

Connecting your account puts your Storytel library inside Music Assistant, with the catalogue there to search. Where you got to in a book travels in both directions, so a title started in the Storytel app can be picked up again in Music Assistant, or the other way round.

> [!NOTE]
> A valid Storytel subscription and account are required to add this provider.

## Features

|                                                   |                      |
| :------------------------------------------------ | :------------------: |
| Subscription FREE                                 |          No          |
| Self-Hosted Local Media                           |          No          |
| Media Types Supported                             | Audiobooks, Podcasts |
| [Recommendations](/ui/#view---discover) Supported |         Yes          |
| Lyrics Supported                                  |          No          |
| [Radio Mode](/ui/#track-menu)                     |          No          |

> | Maximum Stream Quality                             |   Lossy, AAC/MP4     |

Reviewer: ```suggestion
| Artist Top Tracks Support                         |          No          |
| Similar Artists Support                           |          No          |
| Similar Tracks Support                            |          No          |
| Maximum Stream Quality                            |   Lossy, AAC/MP4     |
```

| Media Types Supported                             | Audiobooks, Podcasts |
| [Recommendations](/ui/#view---discover) Supported |         Yes          |
| Lyrics Supported                                  |          No          |
| [Radio Mode](/ui/#track-menu)                     |          No          |
| Artist Top Tracks Support                         |          No          |
| Similar Artists Support                           |          No          |
| Similar Tracks Support                            |          No          |
| Maximum Stream Quality                            |   Lossy, AAC/MP4     |
| Login Method                                      |       Password       |

### Other

- Storytel supports both audiobooks and podcasts from the user's library.
- Search is available for both audiobooks and podcasts.
- Personalized recommendations are supported.
- Chapter navigation.
- Listening progress is automatically synced back to Storytel through bookmarks.
- Library management is supported for adding and removing audiobooks and podcasts.
- The provider can be configured with one or more content languages to influence search and recommendations.

> ### Configuration

Reviewer: ```suggestion
## Configuration
```

To set up the Storytel provider, follow these steps:

1. Navigate to `Settings`.
2. Under Music Sources, click `Add a music source`, select `Storytel`, and fill in your `Username or email` and `Password`.
3. Select the content languages you want to use for search and recommendations.
4. Click `Save`.

### Settings

In addition to `Username or email` and `Password`, there is also:

- <b>Content Languages.</b> Select which languages to include in recommendations and search results. Multiple languages can be selected.

## Known Issues / Notes

> - Browse support is not currently implemented for Storytel. Instead, use the audiobook and podcast libraries to access your media.

Reviewer: ```suggestion
- Browse support is not currently implemented for Storytel. Instead, use the audiobook and podcast libraries to access your media or use search.

```
